challenge01/solution-josebrodriguez.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# solution-josebrodriguez.sh
+
+# Created by: Jose Rodriguez on 10/04/2021
+
+FILENAME="input.txt"
+SED="sed -nE "i
+KEY="UNIX"
+DECRYPTED1=''
+DECRYPTED2=''
+
+read_line(){
+  ID=$1
+
+  A=$(cat $FILENAME | sed -nE "s/($ID: )([A-Z+])/\2/p")
+  echo "$A"
+}
+
+
+decrypt_text(){
+  KEY_CHAR=$1
+  ENC_CHAR=$2
+  
+  ALPHABET="ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+  LETTER_BEFORE=''
+
+  for i in $(seq 1 ${#ALPHABET}); do
+    if [ ${ALPHABET:i-1:1} == $KEY_CHAR ]; then
+      LETTER_BEFORE=$LETTER_BEFORE${ALPHABET:i-2:1}
+    fi
+  done
+
+  echo $ENC_CHAR | tr "[$KEY_CHAR-ZA-$LETTER_BEFORE]" "[A-Z]"
+}
+
+decrypt_loop(){
+  KEY=$1
+  IDLINE=$2
+  DEC_STRING=$3
+  j=0
+  for i in $(seq 1 ${#IDLINE});do
+    j=$(echo "$j+1" | bc)
```

Comment:
We can change this to:

```
j=$(($j+1))
```

The CI Docker image does not have `bc`, so just use internal shell arithmetic.